packages/engine-ui/src/components/CreateVehicle.js

```diff
@@ -52,6 +53,22 @@ const CreateVehicle = ({ onSubmit }) => {
               end: moment(formState.timewindow.end).format('HH:mm'),
             }
           : formState.timewindow,
+      capacity: {
+        weight: parseInt(formState.capacity.weight) || null,
```

Comment:
Are you sure that this cannot be a float?
Same goes for volume

Reply:
the java functions want Integers for majority of settings you can configure

Reply:
Yes, but as the input is in KG, conversion should be made somewhere else

Reply:
you mean we should use metric tons and convert to kg somewhere else?

Reply:
Decided to keep this to satisfy engine, and maybe change this in another ticked

Reply:
Well, right now it can't be a float